mmv1/products/workbench/Instance.yaml

```diff
@@ -384,6 +384,27 @@ properties:
               values:
                 - VIRTIO_NET
                 - GVNIC
+            - !ruby/object:Api::Type::Array
+              name: accessConfigs
```

Comment:
we may need to add `default_from_api: true` to the parent level field though. It seems that the API will return values for this block even if it's explicitly specified and it will cause a permadiff in Terraform. Adding `default_from_api: true` should prevent it from happening. I think the current test if failing due to this issue